addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -886,6 +886,36 @@ section, .oe_img_bg, [data-oe-shape-data] {
     }
 }
 
+// Background Images
+.oe_img_bg {
+    background-size: cover;
+    background-repeat: no-repeat;
+
+    &.o_bg_img_opt_repeat {
+        background-size: auto;
+        background-repeat: repeat;
+    }
+    &.o_bg_img_center {
+        background-position: center;
+    }
+
+    // Compatibility <= 13.0, TODO remove?
+    // -----------------------------------
+    &.o_bg_img_opt_contain {
+        background-size: contain;
+        background-position: center center;
+    }
+    &.o_bg_img_opt_custom {
+        background-size: auto;
+    }
+    &.o_bg_img_opt_repeat_x {
+        background-repeat: repeat-x;
+    }
+    &.o_bg_img_opt_repeat_y {
+        background-repeat: repeat-y;
+    }
+}
+
```

Comment:
Isn't there a way to put those in common rather than duplicating them ? If not then I think we should at least remove the compatibility css rules IMO. It makes sense in website because you can have your old website from <= 13.0, but are you really going to send the same marketing email in 2022 than you did in 2020 ? Probably not.